test/builder_test.js

```diff
@@ -15,7 +15,7 @@ var chaiAsPromised = require('chai-as-promised'); chai.use(chaiAsPromised)
 var sinonChai = require('sinon-chai'); chai.use(sinonChai)
 var multidepRequire = require('multidep')('test/multidep.json')
 
-var Plugin = multidepRequire('broccoli-plugin', '1.2.0')
+var Plugin = require('broccoli-plugin')
```

Comment:
Note: update this back to `multidepRequire('broccoli-plugin', '1.3.0') when https://github.com/broccolijs/broccoli-plugin/pull/19 lands.